### PR TITLE
Update doc for screen ruler of powertoys

### PR DESCRIPTION
**Pull request by Sophanatprime**
Update doc for https://github.com/microsoft/PowerToys/pull/35887.

### CODE REVIEW

**Comment by Sophanatprime** [comment]
@microsoft-github-policy-service agree

File: hub/powertoys/screen-ruler.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ From the Settings menu, the following options can be configured:
 | Capture screen continuously during measuring | When off, the utility takes a single snapshot of your screen. When this is turned on, the utility will attempt real-time detection. Continuous mode will consume more resources when in use. |
 | Per color channel edge detection | Test if all color channels are within a tolerance distance from each other. Otherwise, check that the sum of all color channels differences is smaller than the tolerance. |
 | Pixel tolerance for edge detection | A value between 0-255. A higher value will provide a higher variation so it will be more forgiving with things like gradients and shadows. |
+| Extra units of measurement | Except for showing the measurement in pixels, one can also choose another unit. Available values are pixels, inches, centimeteres, and millimeteres. |
```
Comment (alvinashcraft) [inline hub/powertoys/screen-ruler.md]:
```suggestion
| Extra units of measurement | Choose an alternate unit of measure to display in addition to pixels. Available values are pixels, inches, centimeters, and millimeters. |
```

**Review by alvinashcraft — APPROVED** [review]
Looks good. I'll hold on to this PR until the change is part of a PowerToys release.

**Comment by htcfreek** [comment]
@alvinashcraft 
Did you missed to merge this?

**Comment by alvinashcraft** [comment]
> @alvinashcraft Did you missed to merge this?

The linked feature PR must have been merged after I made my docs list for 0.87. It was still open the last time I looked at it. I'll resolve the conflicts here and merge it today.

**Review by alvinashcraft — APPROVED** [review]

**Pull request merged**